<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminClass</class>
 <widget class="QWidget" name="AdminClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>472</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>120</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Administrator Mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="movieListLabel">
         <property name="font">
          <font>
           <family>Century Schoolbook</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Movie List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undoButton">
         <property name="font">
          <font>
           <family>Century Schoolbook</family>
          </font>
         </property>
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redoButton">
         <property name="font">
          <font>
           <family>Century Schoolbook</family>
          </font>
         </property>
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="repoComboBox">
       <property name="font">
        <font>
         <family>Century Schoolbook</family>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Detailed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Short</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="movieListWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="titleLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="genreLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Genre</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="genreLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="likesLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Likes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="likesLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="yearLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yearLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="trailerLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Trailer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="trailerLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="durationLabel">
             <property name="font">
              <font>
               <family>Century Schoolbook</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Duration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="durationLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="font">
            <font>
             <family>Century Schoolbook</family>
            </font>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="font">
            <font>
             <family>Century Schoolbook</family>
            </font>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="font">
            <font>
             <family>Century Schoolbook</family>
            </font>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
